contracts/pallet-ibc/src/client.rs

```diff
@@ -95,7 +95,10 @@ where
 		if cfg!(any(test, feature = "runtime-benchmarks")) {
 			"tendermint".to_string()
 		} else {
-			"beefy".to_string()
+			match <T as Config>::LIGHT_CLIENT_PROTOCOL {
+				LightClientProtocol::Beefy => "beefy".to_string(),
```

Comment:
why aren't we adding tendermint as a variant in this enum?

Reply:
The host client type can either be beefy or grandpa only, the only reason I return the host client type as tendermint in tests and benchmarks, is to skip the block header check that needs to be done when fetching the host consensus state, since parachains cannot fetch their own consensus state we need to verify it from a header but that header needs to be present in frame_system, which is not possible in this context

Reply:
thanks